### PR TITLE
Add Home page with calendar

### PR DESCRIPTION
**Pull request by riittagerhards**
Add calendar component, create Home Page displaying current date and calendar showing dates with diary entries. Get name for the header component from Welcome Page with localstorage.

image

Open to do's for the Home Page: Routing from current date to Add Entry Page and from Calendar to Single Entry Page.

part of #7

### CODE REVIEW

File: src/app/components/Calendar/Calendar.styles.css [hunk 1]
```diff
@@ -0,0 +1,25 @@
+.calendar {
+  background-color: var(--color-primary);
+  width: 100vw;
+  border-radius: var(--border-radius-card);
+  margin: 0;
+}
+
+.Calendar__weekDay {
```
Comment (phlgr) [inline src/app/components/Calendar/Calendar.styles.css:8]:
```suggestion
.calendar__weekDay {
```

File: src/app/components/Calendar/Calendar.styles.css [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+.today::after {
+  visibility: hidden;
+}
+
+.today:hover {
+  background-color: var(--color-effect) !important;
+}
+
+.highlightedDays {
```
Comment (phlgr) [inline src/app/components/Calendar/Calendar.styles.css:22]:
```suggestion
.highlightedDays {
```
Either camelCase or snake_case

File: src/app/components/Calendar/Calendar.tsx [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    day: 1,
+  };
+
+  return (
+    <div className={className}>
+      <Calendar
+        calendarClassName="calendar"
+        calendarTodayClassName="today"
+        minimumDate={minimumDate}
+        customDaysClassName={selectedDates}
```
Comment (phlgr) [inline src/app/components/Calendar/Calendar.tsx:35]: Only render the calendar if the selectedDates are already set.

File: src/app/pages/Home/AddNewEntryIcon.svg [hunk 4]
```diff
@@ -0,0 +1,4 @@
+<svg width="32" height="32" viewBox="0 0 32 32" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M17.74 30L16 29L20 22H26C26.5304 22 27.0391 21.7893 27.4142 21.4142C27.7893 21.0391 28 20.5304 28 20V8C28 7.46957 27.7893 6.96086 27.4142 6.58579C27.0391 6.21071 26.5304 6 26 6H6C5.46957 6 4.96086 6.21071 4.58579 6.58579C4.21071 6.96086 4 7.46957 4 8V20C4 20.5304 4.21071 21.0391 4.58579 21.4142C4.96086 21.7893 5.46957 22 6 22H15V24H6C4.93913 24 3.92172 23.5786 3.17157 22.8284C2.42143 22.0783 2 21.0609 2 20V8C2 6.93913 2.42143 5.92172 3.17157 5.17157C3.92172 4.42143 4.93913 4 6 4H26C27.0609 4 28.0783 4.42143 28.8284 5.17157C29.5786 5.92172 30 6.93913 30 8V20C30 21.0609 29.5786 22.0783 28.8284 22.8284C28.0783 23.5786 27.0609 24 26 24H21.16L17.74 30Z" fill="white"/>
+<path d="M17 9H15V13H11V15H15V19H17V15H21V13H17V9Z" fill="white"/>
+</svg>
```
Comment (phlgr) [inline src/app/pages/Home/AddNewEntryIcon.svg:4]: I would make a React Component out of this

File: src/app/pages/Home/Home.tsx [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+    'APR',
+    'MAY',
+    'JUN',
+    'JUL',
+    'AUG',
+    'SEP',
+    'OCT',
+    'NOV',
+    'DEC',
+  ];
```
Comment (phlgr) [inline src/app/pages/Home/Home.tsx:22]: This is an absolute constant and could be named: `MONTHS`

**Review by phlgr — CHANGES_REQUESTED** [review]

File: src/app/components/Calendar/Calendar.styles.css (hunk 1 above)
Comment (riittagerhards) [reply src/app/components/Calendar/Calendar.styles.css:8]: I want to overrule the default settings of the npm package I use for the calendar and need to take the classname from the package. It looks different than my classnames.

File: src/app/components/Calendar/Calendar.styles.css (hunk 2 above)
Comment (riittagerhards) [reply src/app/components/Calendar/Calendar.styles.css:22]: Unfortunately, I'm using camelCase but npm calendar package comes with snake_case classnames.

**Pull request merged**